src/ui/common/src/components/pages/integration/id/index.tsx

```diff
@@ -158,6 +165,110 @@ const IntegrationDetailsPage: React.FC<IntegrationDetailsPageProps> = ({
     return null;
   }
 
+  let preview = (
+    <Alert severity="error" sx={{ width: '80%' }}>
+        <>
+          We currently do not support listing data in an S3 bucket. But
+          don&apos;t worry--we&apos;re working on adding this feature! If you
```

Comment:
Can we use an m-dash instead of 2 n-dashes here? I think you can do it with `&mdash;`.